### PR TITLE
Indiana AGI structure

### PR DESCRIPTION
**Pull request by MattHJensen**

### CODE REVIEW

**Review by nikhilwoodruff — APPROVED** [review]
I think this is good to merge with the rename. For adding the logic to the leaf nodes, seems like we should name them with the federal names (from the U.S. Code references), plus the state prefix.

File: openfisca_us/variables/gov/states/in/tax/income/in_add_backs.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from openfisca_us.model_api import *
+
+
+class in_add_backs(Variable):
```
[the remark this answers is not in the document]
Comment (MattHJensen) [reply openfisca_us/variables/gov/states/in/tax/income/in_add_backs.py:4]: In this case it is a state convention to call it an `add-back`. See the [Indiana form.](https://forms.in.gov/Download.aspx?id=15123) (image below). If we use the state conventions, then we could create some consistent mapping later. But I'm also happy to switch to `additions` now if you'd prefer. 

image

[the remark this answers is not in the document]
Comment (nikhilwoodruff) [reply openfisca_us/variables/gov/states/in/tax/income/in_add_backs.py:4]: Oh right- in that case this is good to merge. Thanks!

**Comment by MattHJensen** [comment]
Thanks for the review and merge Nikhil!

**Pull request merged**